docs/source/markdown/podman-kube-play.1.md.in

```diff
@@ -44,6 +44,7 @@ A Kubernetes PersistentVolumeClaim represents a Podman named volume. Only the Pe
 - volume.podman.io/uid
 - volume.podman.io/gid
 - volume.podman.io/mount-options
+- volume.podman.io/import-source
```

Comment:
It would be nice to elaborate a bit more on what this annotation does.

Reply:
You're right. I could't find where in the document I should do that. Should I just add a line under the list?

Reply:
Yes, somewhere below where you think it fits.  As discussed, we need to improve the `kube-play` docs in general but this is a good moment to add a nice description for the new annotation.

Reply:
Done